litellm/proxy/proxy_server.py

```diff
@@ -274,12 +274,10 @@ def generate_feedback_box():
     AnthropicResponseUsageBlock,
```

Comment:
I'm a bit confused. What exactly did we change in this file which is required for us?

Reply:
I think, it is my linter, will remove the file from PR.